docs/rm/custom_instructions.md

```diff
@@ -23,7 +23,7 @@ SCFGRI and SCFGWI read and write a value from or to an SSR configuration registe
 | 0000000 | addr  | 00001 | 001    | dest  | OP-CUSTOM1 | SCFGR     |
 | 0000000 | addr  | value | 010    | 00001 | OP-CUSTOM1 | SCFGW     |
 
-SCFGR and SCFGW read and write a value from or to an SSR configuration register. The value in register *rs2* specifies specifies the address of the register as follows: bits 4 to 0 correspond to *ssr* and indicate the SSR to be used, and the bits 11 to 4 correspond to *reg* and indicate the index of the register. SCFGR places the read value in *rd*. SCFGW moves the value in *rs1* to the selected SSR configuration register.
+SCFGR and SCFGW read and write a value from or to an SSR configuration register. The value in register *rs2* specifies specifies the address of the register as follows: bits 4 to 0 correspond to *ssr* and indicate the SSR to be used, and the bits 11 to 5 correspond to *reg* and indicate the index of the register. SCFGR places the read value in *rd*. SCFGW moves the value in *rs1* to the selected SSR configuration register.
```

Comment:
Great thanks! That is exactly where my confusion started and I forgot about this during the mentioned previous work.

Reply:
Yes, it confused me as well when I checked your PR yesterday. But then I also forgot to mention it there...